src/vulkan-renderer/wrapper/device.cpp

```diff
@@ -312,8 +365,7 @@ void Device::insert_debug_marker(const VkCommandBuffer command_buffer, const std
     assert(!name.empty());
     assert(vk_cmd_debug_marker_insert);
 
-    VkDebugMarkerMarkerInfoEXT debug_marker = {};
-    debug_marker.sType = VK_STRUCTURE_TYPE_DEBUG_MARKER_MARKER_INFO_EXT;
+    auto debug_marker = make_info<VkDebugMarkerMarkerInfoEXT>();
```

Comment:
Nice example for Vulkan initializer methods.
I will add this to my TODO list as well.